### PR TITLE
Refactoring to conform to thoughtbot Ruby style guide (Part 2)

### PR DESCRIPTION
**Pull request by jrade-enova**

### CODE REVIEW

**Comment by jrade-enova** [comment]
Looks like the diff is showing all lines due to changing all tabs to spaces for indents :(

File: triangle_facts/triangle_facts.rb [hunk 1]
```diff
@@ -1,58 +1,67 @@
...
+    end
+
+    angles = calculate_angles(@side1, @side2, @side3)
+    angles_prettified = angles.join(",")
+    puts "The angles of this triangle are #{angles_prettified}"
+
+    if angles.include? 90
+      puts "This triangle is also a right triangle!"
+    end
+    puts ""
```
Comment (paulghaddad) [inline triangle_facts/triangle_facts.rb]: You can just use `puts` here. It will return a blank line.

File: triangle_facts/triangle_facts.rb [hunk 2]
```diff
@@ -1,58 +1,67 @@
...
+
+  def calculate_angles(adjacent, opposite, hypotenuse)
+    angleA = solve_for_angle((opposite**2 + hypotenuse**2 - adjacent**2) / (2.0 * opposite * hypotenuse))
+    angleB = solve_for_angle((adjacent**2 + hypotenuse**2 - opposite**2) / (2.0 * adjacent * hypotenuse))
+    angleC = solve_for_angle((adjacent**2 + opposite**2 - hypotenuse**2) / (2.0 * adjacent * opposite))
+
+    [angleA, angleB, angleC]
+  end
+	
+  attr_reader :side1, :side2, :side3
```
Comment (paulghaddad) [inline triangle_facts/triangle_facts.rb]: Put this at the top of the class.

**Comment by paulghaddad** [comment]
@jrade-enova Good revision. You're good to move on!

**Pull request closed without merging**